README.md

```diff
@@ -0,0 +1,197 @@
+
+There, click "Create Topic" and type in whatever name you want to give to the topic we'll be using to read the events from. In our example, we used "test-topic". Keep track of your topic's name (you can always go to that URL to check it), as it will be needed when you're creating your config file.
+
+![creating a pubsub topic][create-topic]
+create a staging location in which the system will store the jars ran by the workers. To do this, you need to enable the Storage API.
+
+After that, go to https://console.cloud.google.com/storage/browser?project=YOUR-PROJECT-ID . Click "Create Bucket". Fill in the appropriate details. As this is an example project, we suggest you pick the cheapest option.
+
+After creating your bucket, you might want to create a folder inside it, to serve as your staging location. You can also just use the bucket's root. To create a folder in your bucket, after you created the bucket, select it in the list and then click "Create Folder".
+
+Your staging location will be:
+
+```bash
+gs://your-bucket-name/the-folder-that-you-might-have-created
+```
+
+![setting staging location][bucket1]
+
+![setting staging location][bucket2]
+
+#### Bigtable
+
+After enabling Bigtable, we'll need to create an instance where our table will be stored. And then create the actual table.
+
+##### Creating an instance
+
+Go to https://console.cloud.google.com/bigtable/instances?project=YOUR-PROJECT-ID , again taking care to change it with your project id. Click "Create instance" and fill the appropriate details. The name is, again, important. We used "test-instance" in our example. We suggest you to select HDD as a storage type as it is much cheaper, and this is for test-purposes only.
+                  
+|                                            |                                            |
+|:------------------------------------------:|:------------------------------------------:|
+| ![creating a bigtable instance][bigtable1] | ![creating a bigtable instance][bigtable2] |
+
+##### Creating a table
+
+We usually do this through an HBase shell inside Google Cloud Shell ([reference](https://cloud.google.com/bigtable/docs/quickstart-hbase)).  
+
+Go to your Google Cloud Dashboard (https://console.cloud.google.com/home/dashboard?project=YOUR-PROJECT-ID) and click on the little terminal symbol on the top right corner. That will boot your Google Shell. Wait a bit and then run the following commands:
+
+```bash
+$  curl -f -O https://storage.googleapis.com/cloud-bigtable/quickstart/GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ unzip GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ cd quickstart
+$ ./quickstart.sh
+```
+
+This will boot the HBase shell. You'll then issue a ```create``` command, followed by 2 quoted arguments, separated by an argument. These are the table name, and the column family. In our case, we used "test-table" as the table name and "cf1" as the column family:
```

Comment:
>  followed by 2 quoted arguments, separated by an argument.

shouldn't this be separated by a comma? I don't think the part about separating is necessary